fpdf/fpdf.py

```diff
@@ -472,26 +489,15 @@ def add_font(self, family, style='', fname='', uni=False):
             # Font already added!
             return
         if (uni):
-            global SYSTEM_TTFONTS, FPDF_CACHE_MODE, FPDF_CACHE_DIR
-            if os.path.exists(fname):
-                ttffilename = fname
-            elif (FPDF_FONT_DIR and
-                os.path.exists(os.path.join(FPDF_FONT_DIR, fname))):
-                ttffilename = os.path.join(FPDF_FONT_DIR, fname)
-            elif (SYSTEM_TTFONTS and
-                os.path.exists(os.path.join(SYSTEM_TTFONTS, fname))):
-                ttffilename = os.path.join(SYSTEM_TTFONTS, fname)
-            else:
+            global FPDF_CACHE
```

Comment:
FYI global should not actually be needed unless you are changing the value, which I don't think you are.